tempo_embeddings/embeddings/vector_database.py

```diff
@@ -146,19 +146,21 @@ def get_existing_collection(self, name: str) -> Optional[Collection]:
             collection = self.client.get_collection(name, embedding_function=self.embedding_function)
             self.active_collection_name = name
             return collection
-        except:
+        except Exception as e:
```

Comment:
Minor remark: catching the generic `Exception` is equivalent to just running `except`. If possible, you would only catch the specific exceptions that can be expected here.